generatebundlefile/data/input_120.yaml

```diff
@@ -0,0 +1,32 @@
+# This info is hardcoded and comes from https://github.com/aws/eks-anywhere-build-tooling
+name: "v1-21-1001"
```

Comment:
Name should match number

Reply:
This name at the moment is irrelevant, but I can change it. The actual bundle name get's imported from the tag of the file in codebuild.

Reply:
Its fine then you can ignore